src/containers/HackerCountdown.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from 'react'
+import Countdown from './Countdown'
+import { db } from '../utility/firebase'
+import { DB_COLLECTION, DB_HACKATHON } from '../utility/Constants'
+
+export default () => {
+  const [start, setStart] = useState(new Date())
+  const [end, setEnd] = useState(new Date())
+
+  function toDateTime(secs) {
+    var t = new Date(1970, 0, 1);
+    t.setSeconds(secs);
+    return t;
+  }
```

Comment:
I was under the impression we weren't using `timestamp` in firebase? only ISO date strings?

Reply:
ah shoot forgot about that convo, I'll fix